core/src/test/java/io/questdb/griffin/wal/WalTableFailureTest.java

```diff
@@ -1077,6 +1077,43 @@ public void testWalTableSuspendResumeSql() throws Exception {
         });
     }
 
+    @Test
+    public void testAlterTableSetTypeSqlSyntaxErrors() throws Exception {
```

Comment:
would be easier to read if:

```java
assertAlterTableTypeFail("alter table " + tableToken.getTableName() + " set", "[52] 'param' or 'type' expected");

...

private static void assertAlterTableTypeFail(String alterStmt, String expected) {
        try {
            compile(alterStmt);
            Assert.fail("expected SQLException is not thrown");
        } catch (SqlException ex) {
            TestUtils.assertContains(ex.getFlyweightMessage(), expected);
        }
    }
```

Reply:
fixed in [509c5d5](https://github.com/questdb/questdb/pull/2947/commits/509c5d5677a0aa8f5bac45ee8878683fc16321fe)